src/templates/headers/includes/head.template.html

```diff
@@ -0,0 +1,26 @@
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+<meta name="keywords" content="[%url_info name:'meta_keywords'/%]"/>
+<meta name="description" content="[%url_info name:'meta_description'/%]"/>
+<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=0"/>
+<meta property="og:image" content="[%if [@config:current_page_type@] eq 'product'%][@config:homeurl@][%asset_url type:'product' id:'[@SKU@]' thumb:'full'/%][%ELSEIF [@config:current_page_type@] eq 'content'%][@config:homeurl@][%ASSET_url type:'content' id:'[@content_id@]' default:'/assets/website_logo.png'%][%/ASSET_url%][%ELSE%][@config:homeurl@]/assets/website_logo.png[%/if%]"/>
+<meta property="og:title" content="[%url_info name:'page_title'/%]"/>
+<meta property="og:site_name" content="[@config:website_name@]"/>
+<meta property="og:type" content="website"/>
+<meta property="og:url" content="[@config:canonical_url@]"/>
+<meta property="og:description" content="[%url_info name:'meta_description'/%]"/>
+[@config:GOOGLE_VERIFICATION_CODE@]
+<title itemprop='name'>[%url_info name:'page_title'/%]</title>
+<link rel="canonical" href="[@config:canonical_url@]" itemprop="url"/>
+<link rel="shortcut icon" href="/assets/favicon_logo.png?[@config:neto_css_version@]"/>
+<link class="theme-selector" rel="stylesheet" type="text/css" href="[%ntheme_asset%]css/app.css[%/ntheme_asset%]" media="all"/>
```

Comment:
should we host app.css and style.css in the normal header so they can edit?

Reply:
:thumbsup: yep, nice save

Reply:
Will having those two stylesheets after font-awesome, jquery-ui, fancybox mess up cascading? As of right now those stylesheets are before the third party ones, but placing them outside the includes template will place them after.

Reply:
I think this would be a good change, ensuring the app/style stylesheets can actually override third party styles.